packages/dds/map/src/mapKernel.ts

```diff
@@ -661,9 +673,15 @@ export class MapKernel {
                         0x2fd /* pendingMessageId does not match */);
                     this.submitMapClearMessage(op, localOpMetadata.previousMap);
                 },
-                getStashedOpLocalMetadata: (op: IMapClearOperation) => {
+                processStashedOpLocalMetadata: (op: IMapClearOperation) => {
+                    if (this.pendingKeys.size !== 0) {
+                        this.clearExceptPendingKeys();
+                        return;
+                    }
+                    this.clearCore(true);
+                    const copy = this.isAttached() ? new Map<string, ILocalValue>(this.data) : undefined;
```

Comment:
@wes-carlson, will this ever not be attached? I don't think we should have generated an op if it wasn't.

Reply:
Yeah we should always be attached here